src/components/Training/course/CourseInfo.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Padding } from 'styled-components-spacing'
+
+import StyledLink from '../../../components/Common/StyledLink'
+import Image from '../../../components/Common/Image'
+import { SectionTitle, BodyPrimary } from '../../../components/Typography'
+import { Col } from '../../../components/grid'
+import SubtitleWithBody from '../../../components/Common/SubtitleWithBody'
+
+// TODO - retrieve from contentful
```

Comment:
is this TODO to be addressed on this PR?

Reply:
either way, `constants` is not a very good name for this :)

Reply:
@antoniocapelo I am not sure it need to be addressed in this PR. I noticed it was hardcoded and it would make sense to control such strings from contentful. We will think of another name; obviously, we are open to your suggestions for names :)

Reply:
We've removed `constants` from the training page & started using Contentful, straight away :+1: 

As for this `CourseInfo` component, I reckon it requires a few more changes to our content model than is necessarily worth doing within this PR. Raised a Pure Tech Debt ticket here - https://trello.com/c/4dulZp2a/456-review-training-course-content-type

But we've changed the name from `constants`. Let us know what you think